pkg/e2e/test/testdata/sqs_source.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: aws-sqs-source
+spec:
+  limits:
+    readBatchSize: 10 # for aws sqs the batch size should be between 1 -  10 - SQS Valid values: 1 to 10. Default: 1
```

Comment:
Although SQS `SendMessageBatch` API supports sending up to 10 messages, it's the limitation that our user-defined SQS source needs to deal with. User should be able to specify higher numbers. This is something we need to look into, as how sqs source can handle large batches. (doesn't need to addressed in the PR though)

Please create a separate issue to track this topic

Reply:
https://github.com/numaproj-contrib/aws-sqs-source-go/issues/2